brainstormforce/astra
# AST-1861 - Blog Achieve layout 2 and layout 3 broken between 769px to 921px

### Description
There are some issues with the Blog Archive layout 2 and layout 3 between 769px and 921px width.

Issue with Layout 2 - Unwanted spacing on the right when Layout 2 is selected.

Check the screencast: https://d.pr/v/g8Xd61

Issue with Layout 3 - The layout is breaking when the container is Boxed / Content Boxed.

This is how it displays between 769px - 921px: https://d.pr/i/V9DTiu.

Check the screencast attached to see how it displays with Boxed container and Full-Width layout.

### Screenshots


### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
**How to reproduce:**

1. Activate the blog pro module and select Blog Layout 2 or Layout 3
2. Check the frontend between 769px - 921px
3. For Layout 3, check separately with Global Container set to Boxed and Full Width (screencast)



### Checklist:
